features/zones/PageNavigation.feature

```diff
@@ -28,19 +27,18 @@ Feature: Zones: Page navigation
     When I click the "$nav-tabs #<RouteName>-tab a" element
     Then the page title contains "<ListViewTitle>"
```

Comment:
I found that adding a short `cy.wait` in between the two navigations here largely stabilizes this test.

On a side note, should we explicitly set `--browser chrome` for the Cypress commands in our package.json so we always run tests in Chrome rather than Electron?

Reply:
Actually, the test is flaky in both headed and headless mode, but seems much more severe in headless mode (where the test passes most of the time).

Now that I think more about this difference, I’ve retried running the tests locally in headless mode but without video turned on and that seems to really impact the flakyness. I found _some_ indication that this could indeed be related (https://stackoverflow.com/questions/71872119/cypress-fails-tests-randomly-in-headless-mode-but-all-passes-all-tests-in-real-b), but it’s not very conclusive.

Reply:
Looking at the CI test runs and its videos, it looks like whenever we hit the home page with the charts, the page kind of freezes for a moment.

I’m changing this test to go to `/zones` instead of `/` to rule out this isn’t related.

Reply:
🤷🏻 

![image](https://github.com/kumahq/kuma-gui/assets/5774638/620832df-a5cb-4dff-8701-25ecc8806979)

Reply:
Another angle seems to be using Cypress video recordings with GitHub Actions runners: https://github.com/cypress-io/cypress/issues/8210

One piece of advice I’ve seen is switching to use Google Chrome as the browser.

If nothing works, I’ll use https://docs.cypress.io/guides/references/troubleshooting#Log-memory-and-CPU-usage to profile hardware usage.

Reply:
Tests passed a few times now so I’m resolving this for the time being.